_posts/en/newsletters/2019-03-19-newsletter.md

```diff
@@ -0,0 +1,277 @@
+      end-user wallets.
+
+    - Bitcoin Core's coin selection updates over the past two releases
+      allow it to produce transactions without change outputs quite
+      often.  These transactions are more efficient and more privacy
+      preserving than transactions with change, but Esplora currently
+      marks them in red as privacy-leaking because they also match the
+      pattern of sweep transactions.
+
+    - Maxwell proposed a useful addition to the privacy analysis would
+      be identifying when a user controlled multiple UTXOs received to
+      the same address but only sent a transaction spending a subset of
+      those UTXOs.  This behavior makes it possible to connect later
+      transactions spending those UTXOs to the earlier transaction,
+      destroying privacy.
+
+    Overall, it's great to see developers building tools that help
+    people identify flaws in their software or their behavior, but it's
+    also important to consider how users will interact with the tool.
+    As Wuille said near the end of the conversation: "I am super happy
+    that there is a decent explorer now for debugging stuff out there,
+    but I'm concerned about making it sound like it's an actual
+    production tool.  I know people will use explorers, and one that
+    gives good information is better than one that confuses everything.
+    But, really, we shouldn't encourage using [it].  If this privacy
+    detection feature causes people to go look up all their transactions
+    because of a gamification like feeling 'oooh let's see how my
+    transaction did here?!', it's probably a net negative. [...] The
+    most important thing to put on a block explorer is 'Warning: looking
+    up your own addresses on a block explorer leaks your privacy to the
+    site operator'."
+
+- **Spy node ban list updated:** some IP addresses have opened an
+  excessive number of connections to other nodes, probably as part of
+  attempts to monitor transaction propagation to determine which nodes
+  originated which transactions.  Other IP addresses run software that
+  signals poor intentions in other ways.  To help node operators refuse
+  connections from those IP addresses, Gregory Maxwell maintains a
+  ban list that can be imported into Bitcoin Core and compatible nodes.
+  There is absolutely no need to use this centralized list---your fully
+  decentralized node will attempt to connect to a diverse enough set of
+  peers that it should establish at least one honest connection---but
+  using this ban list may reduce the amount of traffic you waste on spy
+  nodes and other bad actors.  The list comes in two formats, one for
+  use on the command line with [bitcoin-cli][banlist cli] and one that
+  can be pasted into the debug console of [Bitcoin Core GUI][banlist
+  gui].  The blacklisted IP addresses are banned for one year and
+  Bitcoin Core will remember the bans between restarts, so you only need
```

Comment:
I tried testing this by pasting the content of the file in my debug console, and I received "Error: IP/Subnet already banned (code -23)." I'll ask about this on IRC too, but I wouldn't have expected that response as I've never set a ban list or used the debug console before.

Reply:
So after a bit of debugging it turns out there is a paste limitation on MacOS such that only 474 of the 669 were applied for me. We should point this out and suggest breaking the list into 2.